datajoint/s3.py

```diff
@@ -59,17 +62,23 @@ def exists(self, name):
         logger.debug('exists: {}:{}'.format(self.bucket, name))
         try:
             self.client.stat_object(self.bucket, str(name))
-        except minio.error.NoSuchKey:
-            return False
+        except minio.error.S3Error as e:
+            if e.code == 'NoSuchKey':
+                return False
+            else:
+                raise e
         return True
 
     def get_size(self, name):
         logger.debug('get_size: {}:{}'.format(self.bucket, name))
         try:
             return self.client.stat_object(self.bucket, str(name)).size
-        except minio.error.NoSuchKey:
-            raise errors.MissingExternalFile from None
-
+        except minio.error.S3Error as e:
+            if e.code == 'NoSuchKey':
+                raise errors.MissingExternalFile from None
+            else:
+                raise e
+  
```

Comment:
Just took a quick look and looks like there is a simple whitespace issue.
```suggestion

```
To run the syntax tests locally, see [here](https://github.com/datajoint/datajoint-python/blob/143ba7b55fd292a0737e72d058670bcb5c6db064/local-docker-compose.yml#L89). To run the style tests locally, see [here](https://github.com/datajoint/datajoint-python/blob/143ba7b55fd292a0737e72d058670bcb5c6db064/local-docker-compose.yml#L90). We should probably add a comment/note in `local-docker-compose.yml` to be more clear about this.

Reply:
Sigh I thought I removed it all....